src/features/Application/TwoColumnComponent.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const GridTwoColumn: React.FC = (props) => {
```

Comment:
This could be useful for lots of other pages besides just application. Therefore, consider making this a styled component in the utility directory (or whatever that directory is named). 

We have loads of generic components, like Input, Checkbox, etc.

Reply:
Thanks for the tip! I added it as a styled component in the directory.